### ISSUE TITLE
Field Data Help Broken in Latest Release

### ISSUE THREAD

**Issue by dnwillia-work** [pyansys/pyfluent#1368]
Hello, I found it is completely different from `field_data` in `0.11.0`.
The construction of FieldData changed.
Especially, it show in `help()` function.
e.g.
In `0.11.0`: `help(solver.field_data.get_scalar_field_data)`
```python

Help on method get_scalar_field_data in module ansys.fluent.core.services.field_data:

get_scalar_field_data(surface_name: str, field_name: str, node_value: Union[bool, NoneType] = True, boundary_value: Union[bool, NoneType] = False) -> Dict[int, <built-in function array>] method of ansys.fluent.core.services.field_data.FieldData instance
    Get scalar field data on a surface.
    
    Parameters
    ----------
    surface_name : str
        Name of the surface.
    field_name : str
        Name of the scalar field.
    node_value : bool, optional
        Whether to provide data for the nodal location. The default is ``True``.
        When ``False``, data is provided for the element location.
    boundary_value : bool, optional
        Whether to provide slip velocity at the wall boundaries. The default is
        ``False``. When ``True``, no slip velocity is provided.
    
    Returns
    -------
    Dict[int, np.array]
        Dictionary containing a map of surface IDs to the scalar field.
```

In `0.12.dev`:
```python
Help on _FieldMethod in FieldData object:

FieldData.get_scalar_field_data = class _FieldMethod(builtins.object)
 |  FieldData.get_scalar_field_data(field_data_accessor, args_allowed_values_accessors)
 |  
 |  Methods defined here:
 |  
 |  __call__(self, *args, **kwargs)
 |      Call self as a function.
 |  
 |  __init__(self, field_data_accessor, args_allowed_values_accessors)
 |      Initialize self.  See help(type(self)) for accurate signature.
 |  
 |  ----------------------------------------------------------------------
 |  Data descriptors defined here:
 |  
 |  __dict__
 |      dictionary for instance variables (if defined)
 |  
 |  __weakref__
 |      list of weak references to the object (if defined)
```

I think it is difficult for reading and debugging.

_Originally posted by @RebelYoung in https://github.com/pyansys/pyfluent/issues/1285#issuecomment-1396596762_

**Comment by seanpearsonuk** [comment]
@rebelyoung @dnwillia-work: this issue was fixed by @hpohekar in [v0.12.dev13](https://github.com/pyansys/pyfluent/releases/tag/v0.12.dev13) 

-> https://github.com/pyansys/pyfluent/pull/1235

Please can you check?

**Comment by dnwillia-work** [comment]
Sean, I guess that should be fixed in the latest released version?  i.e. 0.12.3?

**Comment by RebelYoung** [comment]
> @RebelYoung @dnwillia-work: this issue was fixed by @hpohekar in [v0.12.dev13](https://github.com/pyansys/pyfluent/releases/tag/v0.12.dev13)
> 
> -> #1235
> 
> Please can you check?

I have checked the help() function in [0.12.3].
It is as same as the description in the issue. No changes.

**Comment by hpohekar** [comment]
- is this broken in the current code/latest released version? - yes

- was it fixed by your PR? - yes
